### DISCUSSION TITLE
Using the asset QR codes to move assets to different locations or parent assets

### DISCUSSION THREAD

**Discussion by cn-ml** [hay-kot/homebox#882] [Ideas]
It would be really helpful if one could use the QR codes (maybe printed as stickers on the assets / shelves / rooms) to move items around without having to select a new location manually.

I imagine two different workflows:

1. Moving assets to a new location by scanning the asset and then scanning the new location on the asset page.
2. Inserting multiple assets into a location by scanning those assets with a scanner tool on the location page.

**Comment by markvdb** [comment]
See also https://github.com/hay-kot/homebox/issues/605 .
